modules/installation/proc-config-rhdh-custom-resource.adoc

```diff
@@ -1,4 +1,53 @@
 [id="proc-config-rhdh-custom-resource_{context}"]
 = Configuring the {product-short} Custom Resource
 
-Updates to the Backstage custom resource (CR) are automatically handled by the {product} Operator. However, updates to resources referenced by the CR, such as ConfigMaps or Secrets, are not updated automatically unless the CR itself is updated. If you want to update resources referenced by the CR, then you must manually delete the Backstage Deployment so that the Operator can re-create it with the updated resources.
\ No newline at end of file
+You can change the configuration of the Custom Resource (CR) to program the way that the Operator manages your {product} instance.
+
+[NOTE]
+====
+Updates to the Backstage CR are automatically handled by the {product} Operator. However, updates to resources referenced by the CR, such as ConfigMaps or Secrets, are not updated automatically unless the CR itself is updated. If you want to update resources referenced by the CR, then you must manually delete the Backstage Deployment so that the Operator can re-create it with the updated resources.
+====
+
+.Prerequisites
+* You have an administrative role in {ocp-brand-name}
+* You have added a custom application configuration file to your {ocp-short} instance
```

Comment:
Not sure I understand this prerequisite. Isn't that what users are trying to achieve with the procedure below?

Reply:
@rm3l  Based on our discussion in Slack, I thought we determined that it makes more sense to add the custom app config file to OCP BEFORE configuring the CR. I still plan to change the order in which these two steps are presented in the doc (unless you advise against that), but we can remove this as a preq either way, if it is not actually required / highly recommended that this step occur first.

Reply:
> Based on our discussion in Slack, I thought we determined that it makes more sense to add the custom app config file to OCP BEFORE configuring the CR.

Ah yes, you're right! I found this prereq was conflicting with step 1 ("Add a custom application configuration file to OCP") of the procedure, and that's why I was confused.
The order we discussed (custom app config file first, then configuring the CR) makes sense.

> I still plan to change the order in which these two steps are presented in the doc so that Adding the custom app config file occurs before Configuring CR (unless you advise against that)

+1 for this order.

> but we can remove this as a prereq either way, if it is not actually required / highly recommended that this step occur first.

Is this section only about the initial deployment of RHDH? Or also further customization of an existing RHDH instance? Or both?
If we are talking about the initial deployment of RHDH, there won't be any existing custom application configuration file. Even for a customization of an existing instance, even if the custom application configuration file exists, its content might be as expected already.
So I would suggest removing this as prereq. WDYT?

Reply:
Based on the existing content published in the Admin guide, it sounds like the first step to _Configuring the RHDH CR_ is adding a custom app config file to OCP. Is that true? If so, since Adding a custom app config file to OCP is a separate procedure, it seems like we should tell users to do that procedure first and link to it in the Prerequisites section. 

Also we currently have "Add a custom app config file to OCP..." as the first step of the _Configuring the RHDH CR_ procedure, but it probably shouldn't be. It should probably just be a Prereq instead. 

But I have to admit that I still lack a lot of technical knowledge about this whole process and might be missing or misunderstanding some key or complex parts.

Reply:
PS - In a future PR, I'm also planning to move these sections out from under the Deploying RHDH on OCP section so that they come first in the user journey before the user even tries to Deploy anything.

Reply:
> Based on the existing content published in the Admin guide, it sounds like the first step to _Configuring the RHDH CR_ is adding a custom app config file to OCP so you can reference it in the CR file. Is that true? If so, since Adding a custom app config file to OCP is a separate procedure, it seems like we should tell users to do that procedure first and link to it in the Prerequisites section.
> 
> Also we currently have "Add a custom app config file to OCP..." as the first step of the _Configuring the RHDH CR_ procedure, but it probably shouldn't be. It should probably just be a Prereq instead. And step 1 should by solely focused on editing the CR to reference the custom app config file that must be created before you can even do this procedure. WDYT?
> 

Yes, that makes sense. Users need to create the custom ConfigMap (and Secret too) first, then reference them in the CR, even for the very first deployment of RHDH.